tests/parallel.py

```diff
@@ -15,17 +15,28 @@
 """Unittests for Parallel Layer."""
 import os
 import unittest
-import numpy as np
+
 import jax.numpy as jn
+import numpy as np
 
 import objax
-
+from objax.typing import JaxArray
 
 # Split CPU cores into 8 devices for tests of objax.Parallel
+
```

Comment:
why adding extra blank line?
the comment above directly related to the line `os.environ ....`, in such case usually there is not blank line between comment and the code

Reply:
I changed this file accidentally. Removing all changes.